### DISCUSSION TITLE
Markdown Editor: Show all links as wiki links

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#793] [Feature Requests]
Editing wiki links is much much easier than remembering the file path of the other note. Maybe we could always display all links in the wiki style, and even convert them into wiki style, when editing.

This, again, should be configurable

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by alexjj** [comment]
Using [[wiki links]] would be useful for using [obsidian](obsidian.md) with gitjournal.

**Comment by vHanda** [comment]
> Using [[wiki links]] would be useful for using [obsidian](obsidian.md) with gitjournal.

I just implemented wiki links support in an effort to be compatible with Obsidian - https://twitter.com/GitJournal_io/status/1281145238107697152 . I'll probably make a new release tomorrow. I want to test it out a bit more.

**Comment by LeonardoGentile** [comment]
Hello, was this ever pushed?

**Comment by vHanda** [comment]
> Hello, was this ever pushed?

The basic wiki links was, but automatically converting normal markdown links into wikilinks was not. It landed up be way harder, and needs a lot of changes in GitJournal.
